frontend/src/modules/entitieslist/components/Entity.css

```diff
@@ -102,3 +102,13 @@
 .entity.warnings .status:before {
     color: #ffa10f;
 }
+
+/* button styling for entities sibling */
+
+.entity .noButton {
+    display: none;
+}
```

Comment:
I guess this is temporary?

We shouldn't need this rule, because we don't use the `.noButton` class. We'll show/hide the button using React.

Reply:
Yes, this temporary, show would you use react to show or hide an element?

Reply:
Ignore the last statement I figured it out :)